### PR TITLE
Migrate xcode 11.4 Frameworks

### PR DESCRIPTION
**Pull request by bruno-garcia**
@jmagman I followed the suggested [migration from your comment](https://github.com/flutter/crashy/pull/8#issuecomment-623628503).

Please let me know if I'm missing something

### CODE REVIEW

File: ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -1,116 +1,121 @@
 {
   "images" : [
     {
-      "size" : "20x20",
```
Comment (jmagman) [inline ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json:4]: Is there an actual change in this file?  Looks like the key order just got rearranged.

Comment (bruno-garcia) [reply ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json:4]: True. It was some tooling. I'll just reset the file and reload xcode to make sure it won't mess it up again and cause another dirty file

**Comment by bruno-garcia** [comment]
@jmagman I've undone the change so we could go ahead with this.

I noticed though, if I open `Runner.xcworkspace` and run the build, xcode changes the file:

```
        modified:   ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json
```

**Comment by jmagman** [comment]
> @jmagman I've undone the change so we could go ahead with this.
> 
> I noticed though, if I open `Runner.xcworkspace` and run the build, xcode changes the file:
> 
> ```
>         modified:   ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json
> ```

Xcode gonna Xcode.  I guess let it rearrange the keys if that's what it wants to do.  🤷‍♀️

**Comment by bruno-garcia** [comment]
Lol. OK pushing it back in.

xcode is happy now

**Pull request merged**